doc/tech/policies.md

```diff
@@ -0,0 +1,61 @@
+
+Solving the problems associated with implementing the Abstraction Principle is a work in progress. CNTT has not solved all of the associated problems, nor has the industry. As technology evolves and more designs implement Cloud Native concepts, these problems will be addressed. This document is specifically intended as a one-stop-shop to capture CNTT policies which manage these situations as they exist today and their associated exceptions and transitions, and help drive the technology around and supporting the maturation of the Abstraction Principle.
+
+Several specific technology areas have been identified by CNTT as using an ABI impacted by the Abstraction Principle, as follows:
+- SR-IOV
+- GPU/NPU
+- FPGA/Other Acceleration
+- CPU Instruction Sets and Extensions
+
+The preceding list is not exhaustive; technologies will be added as required.
+
+<a name="7.1.3"></a>
+### 7.1.3 Current CNTT Policies
+
+The following sets of compromises and transition plans comprise the policy for each technology subject to this document.
+
+Be aware the compromises and transition plans contained herein, are directly related to factors which are subject to change with the evolution of technology, with changes in industry direction, with changes in standards, etc. Hence, **the policies are subject to change without notice**, and the reader is advised to consult the latest online Github revision of this chapter. All locally stored, printed or other copies should be considered obsolete.
+
+> **Note to Authors:** Status should be set to "Proposed" when initial content is entered. Once alignment is attained following vetting and discussion, status should be set to "Aligned". Immediately prior to merge, status should be set to "In Force". When amending previously approved language, status should be changed from "In Force" to "In Force (Pending Changes)", followed by "Aligned" and ultimately, "In Force".
+
+**SR-IOV**
+
+SR-IOV transgresses the CNTT Abstraction Principle, which prohibits hardware-dependent code be required in a workload. Refer to the Abstraction Principle in RM Chapter 1 for the related rationale and details. However, CNTT recognizes that today, SR-IOV provides a capability to increase throughput over network interfaces at an economical cost, that some applications as currently implemented, are highly dependent upon. As such, the CNTT approach to SR-IOV shall be:
+
+1. At the present Time Point (TP), TP x, to enable VNFs currently dependent on SR-IOV to transition to Cloud Native technology, VNFs using SR-IOV will PASS OVP qualification, but generate a WARNING indicating the functionality will be deprecated, and to transition to a Cloud Native implementation
+1. At TP x+y, VNFs using SR-IOV will PASS OVP qualification, but generate a WARNING indicating the functionality is deprecated and they must transition to a Cloud Native implementation
```

Comment:
y depends on when a technology is developed to replace SR-IOV to provide the same low latency access to (network interface) devices.

Reply:
Exactly! That's why we specified "y" as a variable, and not as an absolute. Also, that's an example of how to use the framework. It was expected the actual SRIOV policy would be refined, if needed.

Reply:
I think we should describe this somewhere. it is important not just to set fixed dates, but set dates relative to the evolution of technology.